.github/workflows/backend-dev-cd.yml

```
@@ -1,10 +1,15 @@
 name: Backend CD to EC2 for dev
 
+#on:
+#  push:
+#    branches: [ "develop-be" ]
+#    paths:
+#      - 'backend/**'
+#  workflow_dispatch:
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)